src/providers/fault-count/fault-count.ts

```diff
@@ -81,7 +102,65 @@ export class FaultCountProvider {
     return 0;
   }
 
-  sumManoeuvreFaults = (manoeuvres: CatBUniqueTypes.Manoeuvres, faultType: CompetencyOutcome): number => {
+  public sumVehicleCheckFaultsCatBE = (vehicleChecks: CatBEUniqueTypes.VehicleChecks): number => {
+    // TODO: Use the Vechicle checks scoring logic provider, to be done in MES-3737
+    return 0;
+  }
+
+  public getDrivingFaultSumCountCatBE = (data: CatBEUniqueTypes.TestData): number => {
+
+    // The way how we store the driving faults differs for certain competencies
+    // Because of this we need to pay extra attention on summing up all of them
+    const { drivingFaults, manoeuvres, vehicleChecks } = data;
+
+    const drivingFaultSumOfSimpleCompetencies =
+      Object.values(drivingFaults).reduce((acc, numberOfFaults) => acc + numberOfFaults, 0);
+
+    const result =
+      drivingFaultSumOfSimpleCompetencies +
+      this.sumManoeuvreFaults(manoeuvres, CompetencyOutcome.DF) +
+      this.sumVehicleCheckFaultsCatBE(vehicleChecks);
+
+    return result;
+  }
+
+  public getSeriousFaultSumCountCatBE = (data: CatBUniqueTypes.TestData): number => {
```

Comment:
CatBUniqueTypes  -> Cat**BE**UniqueTypes

Reply:
Have another sub-task to update this when https://github.com/dvsa/mes-mobile-app/pull/920 goes in..